x-sounds-mac/AppStorageKeys.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import Combine
+import SwiftUI
+
+enum AppStorageKeys: String {
+    case slackTeamId
+    case slackChanelId
+    case slackCommandEntry
+    case relativeVolumeShouldUse
+    case relativeVolumeValue
+    case FMinstancePath
```

Comment:
Mixed case again. Shouldn't this be
```suggestion
    case fmInstancePath
```
instead?